Builds/CMake/CMakeFuncs.cmake

```diff
@@ -511,20 +511,28 @@ endmacro()
 
 macro(setup_build_boilerplate)
   if (NOT WIN32 AND san)
-    add_compile_options(-fsanitize=${san} -fno-omit-frame-pointer)
+    STRING(REPLACE ";" "," comma_san "${san}")
+    add_compile_options(-fsanitize=${comma_san} -fno-omit-frame-pointer)
```

Comment:
Is there a reason we can't spec to pass a comma on the command line instead of the semicolon?

Reply:
Yes. There are places where we want to treat `${san}` like a list and iterate through its individual values - so we need a semi-colon. There are other places where we need to separate them with a comma because that's how the compiler wants them. So if we passed them with a comma on the command line we'd have to replace that comma with a semi-colon in other places.